crates/virtual-node/src/lib.rs

```diff
@@ -217,7 +217,11 @@ impl VElement {
     pub fn create_element_node(&self) -> CreatedNode<Element> {
         let document = web_sys::window().unwrap().document().unwrap();
 
-        let element = document.create_element(&self.tag).unwrap();
+        let element = if let Some(ns) = html_validation::get_namespace(&self.tag){
```

Comment:
We'll want a new test in the `create_element` tests that shows that this works for an SVG tag.

And also one that shows that anchor tags still work

https://github.com/chinedufn/percy/blob/bf2eb091dccacbd3995777cf94f16de86bf247aa/crates/virtual-dom-rs/tests/create_element.rs

Reply:
BTW, using the create_element for svg elements still attached the svg and its children to the DOM, but it somehow not shown( or misbehaving) in the browser.